ls1intum/Artemis
# `Development`: Drop feedback_conflict table after removal of related feature

> **Warning**
> **DO NOT DEPLOY to a test server!**

> **Warning**
> **Only merge after #6769 is merged!**

### Checklist
#### General



### Motivation and Context


After the feedback conflict feature is removed in #6769, the corresponding database table can be removed.
To be able to test #6769 on test servers without modifying their database, this change is split into a separate PR.

### Description

Remove the `feedback_conflict` table from the database.

### Steps for Testing

1. Create a backup / dump of your local database
2. Start the server
3. Observe that the `feedback_conflict` table is removed
4. Observe that there are no other changes to the database

### Review Progress






#### Code Review
